__tests__/app.test.js

```diff
@@ -40,6 +40,33 @@ describe("GET /api", () => {
   });
 });
 
+describe("GET /api/article/:article_id", () => {
```

Comment:
- A test case worth considering - what if the user requested an article of 9999 or 0? (That doesn't exist)